Realm.XamarinAndroid/packages.config

```diff
@@ -1,4 +1,16 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
   <package id="DotNetCross.Memory.Unsafe" version="0.2.2" targetFramework="monoandroid60" />
+  <package id="Remotion.Linq" version="2.1.1" targetFramework="MonoAndroid60" />
```

Comment:
We actually get a warning about this when building, but it's sort of benign, seeing as how we don't actually use any Android APIs.